native/lambda_game_engine/src/game.rs

```diff
@@ -540,3 +562,23 @@ fn apply_zone_effects(
         });
     }
 }
+
+fn nearest_player(
```

Comment:
nit: This is returning a `Position` so might make more sense to name `nearest_player_position`